src/Domain/FundRepository.php

```diff
@@ -214,6 +214,31 @@ public function findForCampaignsClosedSince(DateTime $closedBeforeDate, DateTime
         return $result;
     }
 
+    /**
+     * @param DateTime $openAtDate Typically now
+     * @return Fund[]
+     */
+    public function findForCampaignsOpenAt(DateTime $openAtDate): array
+    {
+        $query = <<<EOT
+            SELECT fund FROM MatchBot\Domain\Fund fund
+            JOIN fund.campaignFundings campaignFunding
+            JOIN campaignFunding.campaigns campaign
+            WHERE
+                campaign.startDate < :openAtDate AND
+                campaign.endDate > :openAtDate
+            GROUP BY fund
+        EOT;
+
+        /** @var Fund[] $result */
+        $result = $this->getEntityManager()->createQuery($query)
+            ->setParameter('openAtDate', $openAtDate)
+            ->disableResultCache()
```

Comment:
Is there a particular reason for disableResultCache here? Does this query need up to date results more so than other queries?

Reply:
Just trying to maintain similarity with the closest existing analogue, until we pick up the task to look more closely at caches and whether it might actually be doing anything useful.